poker-pals/src/components/pokerComponents/RaiseToggle.vue

```diff
@@ -3,8 +3,8 @@
         <div class="raiseButtonOuter" id="buttonMinus" v-on:click="decrementRaise()">
             <div class="raiseButton centerText">-</div>
         </div>
-        <div class="raiseValue" id="raiseValue"> {{ bigBlind }}</div>
-        <input class="raiseScroll" id="slider" type="range" v-bind:min="bigBlind" v-bind:max="max" v-bind:value="bigBlind"> <!--TODO: set these using props-->
+        <div class="raiseValue">{{ raiseValue }}</div>
+        <input class="raiseScroll" id="slider" type="range" v-bind:min="bigBlind" v-bind:max="bigBlind*20" v-bind:value="raiseValue"> <!--TODO: set these using props-->
```

Comment:
Not sure if this another one of those magic number cases 

bigBlind*20 

Note: it was not brought up during the last pull request.

Reply:
nice catch